client/src/relatedProducts/RelatedProductsView/RelatedProductsList.jsx

```diff
@@ -4,37 +4,65 @@ import RelatedProductsCard from './RelatedProductsCard.jsx';
 
 
 const RelatedProductsList = (props) => {
+  let relatedItemsList = props.allProps.map((product, index) => {
+    return <div key={index}>
+      <RelatedProductsCard
+        key={product.itemId}
+        name={product.itemName}
+        category={product.itemCategory}
+        originalPrice={product.originalPrice}
+        salePrice={product.salePrice}
+        photo={product.photoUrl.thumbnail_url}
+        id={product.itemId}
+        features={product.features}
+        handleProductChange={props.handleProductChange}
+        handleCompareItems={props.handleCompareItems}
+        starRating={product.starRating}
+       />
+     </div>
+  })
+  let prevButton;
```

Comment:
just food for thought - to save space, you could filter the prevButton and nextButton functionality to your helpers.  You can pass the helper function needed from main component to use here... or you can pass a function from your main component to call this functionality in helper

Reply:
Ahh that is a good plan. I feel like I always write out everything in components at first, before eventually getting annoyed enough at seeing all the code to hide it all away in helpers